### PR TITLE
test: add test cases for mountpoint_test.go and portbindngs_test.go

### PR DESCRIPTION
**Pull request by sanmaopep**

### CODE REVIEW

File: apis/opts/portbindings_test.go [hunk 1]
```diff
@@ -69,10 +75,59 @@ func TestVerifyPortBinding(t *testing.T) {
 		wantErr bool
 	}{
 		// TODO: Add test cases.
-		{name: "testCase1", args: args{portBindings: map[string][]types.PortBinding{"21/ftp": {types.PortBinding{HostIP: "", HostPort: "21"}}}}, wantErr: false},
-		{name: "testCase2", args: args{portBindings: map[string][]types.PortBinding{"65537/tcp": {types.PortBinding{}}}}, wantErr: true},
-		{name: "testCase3", args: args{portBindings: map[string][]types.PortBinding{"0/tcp": {types.PortBinding{}}}}, wantErr: false},
-		{name: "testCase4", args: args{portBindings: map[string][]types.PortBinding{"80/http": {types.PortBinding{}}}}, wantErr: false},
+		{
+			name: "testCase1",
```
Comment (ZYecho) [inline apis/opts/portbindings_test.go]: could the test name be more specific? such as `testHttpPortBinding`?

File: apis/opts/portbindings_test.go [hunk 2]
```diff
@@ -69,10 +75,59 @@ func TestVerifyPortBinding(t *testing.T) {
 		wantErr bool
 	}{
```
Comment (ZYecho) [inline apis/opts/portbindings_test.go]: Maybe you could try to add more testcase in this pr~ ^-^

**Comment by ZYecho** [comment]
LGTM
wait for CI pass

**Comment by allencloud** [comment]
Thanks for your great work. 
@sanmaopep 
Wish you have a great journey to participant in community. 
In addition, we have a dingtalk group which contains almost all the develops and contributors of PouchContainer. Would you like to join us? 

ladpdgq9qzzuft3na97nau4_750_990

**Pull request merged**